processor/pom.xml

```diff
@@ -92,6 +92,17 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <MapStruct-Generated-Comment>${project.groupId}:${project.artifactId}:${project.version}</MapStruct-Generated-Comment>
```

Comment:
> which should already be in there.

I checked this out when I started with this PR. Our current manifest is pretty empty:
```java
Manifest-Version: 1.0
Archiver-Version: Plexus Archiver
Built-By: gunnar
Created-By: Apache Maven 3.0.4
Build-Jdk: 1.8.0_25
```
We could add: `<addDefaultImplementationEntries>true</addDefaultImplementationEntries>`. In that case the `MANIFEST.MF` will be extended with:
```java
Implementation-Title: ${project.name}
Implementation-Version: ${project.version}
Implementation-Vendor-Id: ${project.groupId}
Implementation-Vendor: ${project.organization.name}
Implementation-URL: ${project.url}
```
So, we could then use: `Implementation-Version`, extended with the `Build-Jdk` as proposed above in response to a remark of @agudian. WDYT?